sympy/printing/str.py

```diff
@@ -981,19 +981,55 @@ def _print_AppliedBinaryRelation(self, expr):
 
 @print_function(StrPrinter)
 def sstr(expr, **settings):
-    """Returns the expression as a string.
-
-    For large expressions where speed is a concern, use the setting
-    order='none'. If abbrev=True setting is used then units are printed in
-    abbreviated form.
+    """
+    Returns the expression *expr* as a string.
```

Comment:
Am I right in thinking that the purpose of `sstr` is that `sstr(expr)` is the same as `str(expr)` but `sstr` allows controlling the output with optional parameters? If so maybe that could be clarified here.

Reply:
Maybe a see also for `init_printing` would be good since this is basically like the effect that `init_printing` has on `str` except that it doesn't change the global printing settings.

Reply:
Good points! Didn't know that init_printing did that. Just been using it for LaTeX and in rare cases for pretty printing.